bin/cron/export_mysql_database_to_redshift

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+require_relative '../../lib/cdo/only_one'
+exit unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+exit unless rack_env?(:production) && CDO.dashboard_hostname == 'studio.code.org'
+
+require 'aws-sdk-databasemigrationservice'
+require 'cdo/chat_client'
+
+# Export production Aurora MySQL tables to Redshift once a day.
+# 1) Start Data Migration Service (DMS) Replication Tasks, which are configured to drop existing tables in a parallel
+# set of `import_` Redshift schemas and then carry out full load of Aurora MySQL tables into the temporary `import_` schemas.
```

Comment:
Where does this configuration happen?

Reply:
Exporting to temporary tables/schemas is not implemented/configured yet.